environment.yml

```diff
@@ -7,4 +7,5 @@ dependencies:
   - pip:
     - pytest==5.0.1
     - pytest-cov
-    - teamcity-messages
\ No newline at end of file
+    - teamcity-messages
```

Comment:
it might be good to specify the concrete revision needed.

Reply:
it's a simple change